_sass/pages/tags.sass

```diff
@@ -26,7 +26,6 @@
         font-weight: 300
         text-transform: uppercase
         text-align: center
-        margin-bottom: 6rem
```

Comment:
Why that?

Reply:
This is style is override by "margin-bottom: 6rem" in the file "_sass/pages/page.sass". So, this tag is redundant.